docs/spec.md

```diff
@@ -250,6 +250,8 @@ values of type `tensor<T>`).
   * `f8E4M3FNUZ` and `f8E5M2FNUZ` types corresponding to the `E4M3` and `E5M2`
     encodings of the FP8 formats described in
     [8-bit Numerical Formats for Deep Neural Networks](https://arxiv.org/abs/2206.02915).
+  * `f8E4M3B11FNUZ` type corresponding to the `E4M3` encoding of the FP8 formats described in
```

Comment:
https://github.com/openxla/stablehlo/actions/runs/4855239494/jobs/8654920963?pr=1448

Markdown lint check failing on this line, longer than 80chars

Reply:
I reflowed the line, how can I rerun linting?

Reply:
The current approach involves having a committer running it for you, which I did :)